### PR TITLE
Update EAP CD to release 15 in xpass templates / streams

### PR DESCRIPTION
**Pull request by luck3y**
Signed-off-by: Ken Wills <kwills@redhat.com>

NOTICE
======

Master branch is closed! A major refactor is ongoing in devel-40.
Changes for 3.x should be made directly to the latest release branch they're
relevant to and backported from there.

### CODE REVIEW

**Comment by luck3y** [comment]
Is the 'master is closed' message still relevant, or should I open a PR for these against master somewhere too?

**Comment by vrutkovs** [comment]
/ok-to-test
Master is still closed, in 4.0 you'd need to update https://github.com/openshift/cluster-samples-operator anyway

**Comment by luck3y** [comment]
@vrutkovs any update on this one?

File: roles/openshift_examples/files/examples/x86_64/xpaas-streams/eap-cd-image-stream.json [hunk 1]
```diff
@@ -9,29 +9,29 @@
...
+    {
             "kind": "ImageStream",
             "apiVersion": "v1",
             "metadata": {
                 "name": "eap-cd-openshift",
                 "annotations": {
                     "openshift.io/display-name": "JBoss EAP continuous delivery",
                     "openshift.io/provider-display-name": "Red Hat, Inc.",
-                    "version": "13.0"
+                    "version": "14.0"
```
Comment (vrutkovs) [inline roles/openshift_examples/files/examples/x86_64/xpaas-streams/eap-cd-image-stream.json]: Use latest tag version here?
```suggestion
                    "version": "15.0"
```

File: roles/openshift_examples/files/examples/x86_64/xpaas-templates/eap-cd-amq-persistent-s2i.json [hunk 2]
```diff
@@ -5,19 +5,19 @@
         "annotations": {
             "iconClass": "icon-eap",
             "tags": "eap,javaee,java,jboss",
-            "version": "1.5.0",
-            "openshift.io/display-name": "JBoss EAP CD + AMQ 7 (with https)",
+            "version": "14.0",
```
Comment (vrutkovs) [inline roles/openshift_examples/files/examples/x86_64/xpaas-templates/eap-cd-amq-persistent-s2i.json]: Same here - latest tag is 15, but version is 14?

Comment (luck3y) [reply roles/openshift_examples/files/examples/x86_64/xpaas-templates/eap-cd-amq-persistent-s2i.json]: @vrutkovs thanks and sorry for wasting your time by missing that.

**Comment by luck3y** [comment]
@vrutkovs Sorry to be a pest, but this should be good to go now.

**Pull request merged**